google-cloud-datastore/src/main/java/com/google/cloud/datastore/testing/LocalDatastoreHelper.java

```diff
@@ -193,14 +193,22 @@ public void start() throws IOException, InterruptedException {
   }
 
   /**
-   * Reset the internal state of the Datastore emulator.
+   * Resets the internal state of the Datastore emulator.
+   *
+   * <p>When running tests, one might {@code reset()} before each test, so earlier tests would not
+   * affect later ones.
    */
   public void reset() throws IOException {
     sendPostRequest("/reset");
   }
 
   /**
    * Stops the Datastore emulator.
+   *
+   * <p>It is important to stop the emulator. Since the emulator runs in its own process, not
+   * stopping it might cause it to become orphan.
+   *
+   * <p>It is not required to call {@link #reset()} before {@code stop()}.
    */
```

Comment:
Can we say anything about duration. 5 seconds seems to work for me, 1 second did not. I guess this is to wait the emulator being stopped to avoid weird test failures on subsequent runs if the emulator got stuck.

Reply:
Sounds good. PTAL